### PR TITLE
docs: fix i18n

### PR DESCRIPTION
**Pull request by zmstone**
Release version: `v/e5.5.1`

## Summary

This is an attempt to sync the changes made in  https://github.com/emqx/emqx-i18n/tree/v55
Also tried to fix some doc bug found during this process.

* HStreamDB -> MySQL (copy-paste error)
* Bridge -> Action

### CODE REVIEW

File: rel/i18n/emqx_bridge_azure_event_hub.hocon [hunk 1]
```diff
@@ -120,63 +69,39 @@ required_acks.label:
...
- - JSON: only add JSON values to Azure Event Hubs headers,
-and encode it to JSON strings before sending."""
+"""The encoding mode for headers.
+
+ - `none`: Add only binary values to Azure Event Hubs headers
+ - `json`: Add only JSON values to headers and encode them into JSON strings before sending."""
 
 kafka_header_value_encode_mode.label:
-"""Azure Event Hubs headers value encode mode"""
+"""Headers value encode mode"""
```
Comment (ieQu1) [inline rel/i18n/emqx_bridge_azure_event_hub.hocon:97]: This is a bit hard to parse. 
```suggestion
"""The encoding mode for the header values."""
```
?

File: rel/i18n/emqx_bridge_hstreamdb_connector.hocon [hunk 2]
```diff
@@ -13,7 +13,7 @@ type.label:
 """Connector Type"""
 
 name.desc:
-"""Connector name, used as a human-readable description of the connector."""
+"""Connector name, a human-readable identifier."""
```
Comment (ieQu1) [inline rel/i18n/emqx_bridge_hstreamdb_connector.hocon:16]:
```suggestion
"""A human-readable identifier of the connector."""
```

File: rel/i18n/emqx_bridge_kafka.hocon [hunk 3]
```diff
@@ -191,7 +179,7 @@ max_batch_bytes.label:
 """Max Batch Bytes"""
 
 required_acks.desc:
-"""Required acknowledgements for Kafka partition leader to wait for its followers before it sends back the acknowledgement to EMQX Kafka producer
+"""he acknowledgement criteria for the partition leader. It determines the level of confirmation required from partition replicas before sending an acknowledgement back to the producer.
```
Comment (ieQu1) [inline rel/i18n/emqx_bridge_kafka.hocon:182]: he -> The

File: rel/i18n/emqx_audit_api.hocon [hunk 4]
```diff
@@ -1,57 +1,61 @@
 emqx_audit_api {
 
 audit_get.desc:
-"""Get audit logs based on filter API, empowers users to efficiently
+"""Get audit logs with filtering parameters. This feature enable users to efficiently
```
Comment (id) [inline rel/i18n/emqx_audit_api.hocon:4]:
```suggestion
"""Get audit logs with filtering parameters. This feature enables users to efficiently
```

File: rel/i18n/emqx_bridge_azure_event_hub.hocon [hunk 5]
```diff
@@ -262,29 +263,31 @@ kafka_message_value.label:
...
+<code>key_dispatch</code>: Assigns messages to partitions based on a hash of the message key,
+ensuring consistent partition for messages with the same key."""
 
 partition_strategy.label:
 """Partition Strategy"""
 
 buffer_segment_bytes.desc:
 """Applicable when buffer mode is set to <code>disk</code> or <code>hybrid</code>.
-This value is to specify the size of each on-disk buffer file."""
+This setting specifies the size of each buffer file stored on disk."""
```
Comment (id) [inline rel/i18n/emqx_bridge_azure_event_hub.hocon:277]:
```suggestion
This setting specifies the size of each buffer file segment stored on disk."""
```
?

File: rel/i18n/emqx_bridge_gcp_pubsub.hocon [hunk 6]
```diff
@@ -10,16 +10,16 @@ desc_config.desc:
...
-"""Bridge name, used as a human-readable description of the bridge."""
+"""Name, used as a human-readable identifier."""
 
 desc_name.label:
-"""Bridge Name"""
+"""Name"""
 
 desc_type.desc:
-"""The Bridge Type"""
+"""The bridge type"""
```
Comment (id) [inline rel/i18n/emqx_bridge_gcp_pubsub.hocon:22]:
```suggestion
"""Action Type"""
```
?

File: rel/i18n/emqx_bridge_gcp_pubsub.hocon [hunk 7]
```diff
@@ -10,16 +10,16 @@ desc_config.desc:
...
-"""GCP PubSub Bridge Configuration"""
+"""GCP PubSub Configuration"""
 
 desc_name.desc:
-"""Bridge name, used as a human-readable description of the bridge."""
+"""Name, used as a human-readable identifier."""
 
 desc_name.label:
-"""Bridge Name"""
+"""Name"""
```
Comment (id) [inline rel/i18n/emqx_bridge_gcp_pubsub.hocon:19]:
```suggestion
"""Action Name"""
```
?

**Review by id — APPROVED** [review]

File: rel/i18n/emqx_bridge_azure_event_hub.hocon (hunk 5 above)
Comment (zmstone) [reply rel/i18n/emqx_bridge_azure_event_hub.hocon:277]: changed to 'segment fie', rather than 'file segment'

**Review by id — APPROVED** [review]
